microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -159,218 +235,325 @@ If you publish a person's Personal Information to an outside service or third-pa
+If the user disables notifications, either on a product-specific or system-wide basis, then your extension must adhere to the user's notification settings and remain functional even though notifications may be disabled.
 
-If your product uses MPNS or WNS to transmit notifications, it must comply with the following requirements:
+You'll be receiving notifications related to your accounts, including but not limited to, login attempts, account changes, and security alerts.  To ensure the security of your accounts and the add-ons you submit, verify your accounts through a secure and approved method.
 
-#### 1.9.1 General Guidance
 
-Notifications provided through WNS or MPNS are considered product content and are subject to all Add-ons Catalog Developer Policies.
+###### 1.9.1 General guidance
 
-#### 1.9.2 Ownership of notifications
+Notifications provided through WNS or MPNS are considered product content and are subject to all Microsoft Edge Add-ons store developer policies.
+
+
+###### 1.9.2 Ownership of notifications
 
 You must not obscure or try to disguise the source of any notification initiated from your extension.
 
-#### 1.9.3 No Confidential or Sensitive Information
+
+###### 1.9.3 No confidential or sensitive information
 
 You must not include in a notification any information users may reasonably consider confidential or sensitive.
 
-#### 1.9.4 Purpose of notifications
 
-Notifications sent from your extension must relate to that extension or to other extensions you publish in Microsoft Edge Add-ons store and must not include promotional messages of any kind that aren't related to your extensions.
+###### 1.9.4 Purpose of notifications
+
+Notifications sent from your extension must relate to that extension or to other extensions you publish in the Microsoft Edge Add-ons store, and must not include promotional messages of any kind that aren't related to your extensions.
 
-### 1.10 Advertising Conduct and Content
+
+<!-- ------------------------------ -->
+#### 1.10 Advertising conduct and content
 
 For all advertising related activities, the following requirements apply:
 
-#### 1.10.1 Purpose
 
-The primary content of your extension must not be advertising, and advertising must be clearly distinguishable from other content in your extension.
+###### 1.10.1 Purpose
+
+The primary content of your extension must not be advertising, and your extension should not bundle other extension offers within the same flow.
+
+Users should not be misled to click on advertisements to fully use the extension.
 
-#### 1.10.2 Policies and Agreements
+
+###### 1.10.2 Policies and agreements
 
 Any advertising content your extension displays must adhere to [Microsoft Advertising Network policies](https://about.ads.microsoft.com/policies/home).
 
 If your extension displays ads, all content displayed must conform to the advertising requirements of the [App Developer Agreement](/legal/windows/agreements/app-developer-agreement) and this Policy.
```

Comment:
what is "this Policy" - 
* these Microsoft Edge Add-ons store developer policies; entire article
* policy section #  1.10.2 (Policies and agreements)

@Bhuvana-Priya